src/network/network_content.cpp

```diff
@@ -56,10 +56,10 @@ bool ClientNetworkContentSocketHandler::Receive_SERVER_INFO(Packet *p)
 	ci->id       = (ContentID)p->Recv_uint32();
 	ci->filesize = p->Recv_uint32();
 
-	p->Recv_string(ci->name, lengthof(ci->name));
-	p->Recv_string(ci->version, lengthof(ci->version));
-	p->Recv_string(ci->url, lengthof(ci->url));
-	p->Recv_string(ci->description, lengthof(ci->description), SVS_REPLACE_WITH_QUESTION_MARK | SVS_ALLOW_NEWLINE);
+	ci->name        = p->Recv_string(32);
+	ci->version     = p->Recv_string(16);
+	ci->url         = p->Recv_string(96);
+	ci->description = p->Recv_string(512, SVS_REPLACE_WITH_QUESTION_MARK | SVS_ALLOW_NEWLINE);
```

Comment:
Oof, more of those magic numbers. Seems this part of the code really collected them ... Well, just see if you can/want to do something with them, or leave them, what-ever ;)